DQN/README.md

```diff
@@ -0,0 +1,35 @@
+## Set Up
+1. Install [conda](https://www.anaconda.com/products/individual)
+2. Install dependencies with `environment.yml`
+    ```
+    conda env create -f environment.yml
+    ```
+    Check the new environment *mario* is [created successfully](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#creating-an-environment-from-an-environment-yml-file).
+
+3. Activate *mario* enviroment
+    ```
+    conda activate myenv
```

Comment:
change to "conda activate mario"

Reply:
When I install matplotlib it says it requires pyglet<=1.5.0,>=1.4.0, although yml file installs pyglet 1.5.5, incompatible

Reply:
Missing dependency skimage

Reply:
Done